release_notes.md

```diff
@@ -0,0 +1,9 @@
+# What's New
+
+- Following new pages have been added. Each page and its options can be configured using the *Outbreak Response Framework* solution pack.
+
+    - **Configure Integration**: Contains separate configuration tab for each integration selected in the *Select Integration* page
+    - **Investigation Schedule**: Automate the investigation frequency and customize the investigation window
+    - **Installation & Notification**: Receive notifications when an outbreak specific solution pack becomes available and automatically install them
+
+- Removed Ticketing/ITSM integration page
```

Comment:
Install Latest Outbreak Response solution pack by clicking the Ingest now button on finish page